app/renderers/universal_viewer.rb

```diff
@@ -0,0 +1,35 @@
+# frozen_string_literal: true
+class UniversalViewer
+  include Singleton
```

Comment:
This is another thing I took pretty much without any questions from Hyku, so couldn't really answer. We could ask Trey or Justin if you'd like, though.